<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Login</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Login</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_usr">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit_usr"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_pw"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_pw">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="button_logout">
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="button_login">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_host">
            <property name="text">
             <string>Hostname:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_db">
            <property name="text">
             <string>Database:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_host"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_db"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Search</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="1">
           <widget class="QLabel" name="label_update">
            <property name="text">
             <string>label</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLineEdit" name="lineEdit_search"/>
          </item>
          <item row="3" column="4">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item row="1" column="5">
           <widget class="QCheckBox" name="checkBox_desc">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Search in:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="button_search">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QPushButton" name="button_update">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QCheckBox" name="checkBox_part_no">
            <property name="text">
             <string>Part number</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="5">
           <widget class="QTableWidget" name="tableWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="columnCount">
             <number>6</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Sl no</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Part Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>MFG P/N</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Location</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantity</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Part Description</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEdit_part"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Enter Sl no:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Admin Console</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Other settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Notify when quantity of an item is less than:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_access">
            <property name="toolTip">
             <string>Takes effect Immediately!</string>
            </property>
            <property name="text">
             <string>Disable access to database for other users</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBox_notify">
            <property name="toolTip">
             <string>(Changes get saved automatically)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Manage Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="0">
           <widget class="QLabel" name="label_filesave">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="button_import">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="button_export">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Manage Users</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Manage Users</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="2">
           <widget class="QPushButton" name="button_revoke">
            <property name="text">
             <string>Revoke</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="button_grant">
            <property name="text">
             <string>Grant</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_usrupd">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QComboBox" name="comboBox_grt"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QComboBox" name="comboBox_rev"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_current_usr">
     <property name="text">
      <string>Logged in as: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_usr</tabstop>
  <tabstop>lineEdit_pw</tabstop>
  <tabstop>lineEdit_host</tabstop>
  <tabstop>lineEdit_db</tabstop>
  <tabstop>button_login</tabstop>
  <tabstop>button_logout</tabstop>
  <tabstop>lineEdit_search</tabstop>
  <tabstop>button_search</tabstop>
  <tabstop>checkBox_part_no</tabstop>
  <tabstop>checkBox_desc</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>button_update</tabstop>
  <tabstop>checkBox_access</tabstop>
  <tabstop>spinBox_notify</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
